Template/Pages/Worship Services.html

```diff
@@ -1,10 +1,14 @@
 <!--
  Title: Worship Services
  -->
-<p>SUNDAY 10:30 AM</p>
-<p>-Worship</p>
-<p>-Children leave for Childrens' Church part way through the service.</p>
-<p>Our Sunday morning worship experience includes singing praise hymns, both old and new, time for corporate and personal prayer and inspiring sermons.</p>
+<p class="center">SUNDAY 10:30 AM</p>
+<p>Our Sunday morning worship experience includes:<p/>
+<ul>
+    <li>singing praise hymns, both old and new</li>
```

Comment:
Is this indented for a reason? HTML folds all whitespace down to a single space, so the indent here does nothing.

Reply:
Just followed the example I found at w3schools to create a bullet list.

Reply:
Indentation to arrange code is fine. It just looked funny because only one out of several `<li>` (**l**ist **i**tem) elements was indented, and I couldn’t tell why.